Source/WebCore/loader/cache/CachedResourceLoader.h

```diff
@@ -233,16 +236,16 @@ class ResourceCacheValidationSuppressor {
 public:
     ResourceCacheValidationSuppressor(CachedResourceLoader& loader)
         : m_loader(loader)
-        , m_previousState(m_loader.m_allowStaleResources)
+        , m_previousState(loader.m_allowStaleResources)
     {
-        m_loader.m_allowStaleResources = true;
+        m_loader->m_allowStaleResources = true;
     }
     ~ResourceCacheValidationSuppressor()
     {
-        m_loader.m_allowStaleResources = m_previousState;
+        m_loader->m_allowStaleResources = m_previousState;
```

Comment:
Should we consider the possibility that `m_loader` has been cleared at this point, since it's a WeakRef and we don't control it?

Reply:
can’t / shouldn’t